starter-code/app.js

```diff
@@ -1,5 +1,64 @@
 const express = require('express');
-const app = express();
-const Chuck  = require('chucknorris-io');
+const expressLayouts = require('express-ejs-layouts');
+const bodyParser = require('body-parser');
+const Chuck = require('chucknorris-io');
```

Comment:
I think you forgot to do --save when you installed this and then you went back and did it later, cause you ended up with 2 package.jsons.  You should delete the wrong one if this happens and make sure the one you're using is called package.json you can't use another name